src/app/routes/api/v3/admin/forms/__tests__/admin-forms.form.routes.spec.ts

```diff
@@ -1429,4 +1437,233 @@ describe('admin-form.form.routes', () => {
       })
     })
   })
+
+  describe('POST /:formId/fields/:fieldId/duplicate', () => {
+    it('should return 200 with updated form with duplicated field', async () => {
+      // Arrange
+      const fieldToDuplicate = generateDefaultField(BasicField.Date)
+      const formToUpdate = (await EmailFormModel.create({
+        title: 'Form to update',
+        emails: [defaultUser.email],
+        admin: defaultUser._id,
+        form_fields: [fieldToDuplicate],
+      })) as IPopulatedForm
+
+      const expectedOriginalField = {
+        ...omit(fieldToDuplicate, ['getQuestion']),
+        _id: new ObjectId(fieldToDuplicate._id),
+      }
+      const expectedDuplicatedField = {
+        ...omit(fieldToDuplicate, ['_id', 'globalId', 'getQuestion']), // do not compare _id and globalId
+      }
+
+      // Act
+      const response = await request.post(
+        `/admin/forms/${formToUpdate._id}/fields/${fieldToDuplicate._id}/duplicate`,
+      )
+
+      // Assert
+      const actual = await EmailFormModel.findById(formToUpdate._id)
+        .populate({
+          path: 'admin',
+          populate: {
+            path: 'agency',
+          },
+        })
```

Comment:
I don't think you need to populate the admin field, because you're not checking any of the admin info, right?